packages/components/addon/components/hds/dialog-primitive/description.hbs

```diff
@@ -0,0 +1,8 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Text::Body class="hds-dialog-primitive__description" @tag="div" @size="200" @color="primary" ...attributes>
+  {{yield}}
+</Hds::Text::Body>
```

Comment:
There had been mention that we may want to make the `Description` available to the `Modal` component as well instead of only using it in the `Flyout` as it is currently so that could effect the styling.

@majedelass any thoughts on this yet?